core/src/test/java/io/questdb/cutlass/line/udp/LineUdpPartitionReadOnlyTest.java

```diff
@@ -0,0 +1,283 @@
+        final String tableName = testName.getMethodName();
+        int numEvents = 5000;
+        SOCountDownLatch sendComplete = new SOCountDownLatch(1);
+        assertServerMainWithLineUDP(
+                tableName,
+                () -> {
+                    long[] timestampNano = {firstPartitionTs, secondPartitionTs, thirdPartitionTs, lastPartitionTs, futurePartitionTs};
+                    try (LineUdpSender sender = new LineUdpSender(NetworkFacadeImpl.INSTANCE, 0, Net.parseIPv4("127.0.0.1"), ILP_PORT, 200, 1)) {
+                        for (int tick = 0; tick < numEvents; tick++) {
+                            int tickerId = tick % timestampNano.length;
+                            timestampNano[tickerId] += lineTsStep;
+                            sender.metric(tableName)
+                                    .tag("s", "lobster")
+                                    .field("l", 88)
+                                    .field("i", 2124)
+                                    .at(timestampNano[tickerId]);
+                        }
+                        sender.flush();
+                    } finally {
+                        sendComplete.countDown();
+                    }
+                },
+                sendComplete,
+                5000,
+                null,
+                false, false, false, true, false
+        );
+    }
+
+    @Test
+    public void testActivePartitionReadOnlyAndO3UnderActivePartitionUDP() throws Exception {
+        final String tableName = testName.getMethodName();
+        int numEvents = 5000;
+        SOCountDownLatch sendComplete = new SOCountDownLatch(1);
+        assertServerMainWithLineUDP(
+                tableName,
+                () -> {
+                    long[] timestampNano = {firstPartitionTs, secondPartitionTs, thirdPartitionTs, lastPartitionTs};
+                    try (LineUdpSender sender = new LineUdpSender(NetworkFacadeImpl.INSTANCE, 0, Net.parseIPv4("127.0.0.1"), ILP_PORT, 200, 1)) {
+                        for (int tick = 0; tick < numEvents; tick++) {
+                            int tickerId = tick % timestampNano.length;
+                            timestampNano[tickerId] += lineTsStep;
+                            sender.metric(tableName)
+                                    .tag("s", "lobster")
+                                    .field("l", 88)
+                                    .field("i", 2124)
+                                    .at(timestampNano[tickerId]);
+                        }
+                        sender.flush();
+                    } finally {
+                        sendComplete.countDown();
+                    }
+                },
+                sendComplete,
+                1666,
+                null,
+                true, false, true, true
+        );
+    }
+
+    @Test
+    public void testTableIsReadOnlyUDP() throws Exception {
+        Assume.assumeFalse(Os.isLinux()); // cannot debug this myself
```

Comment:
This test is still disabled. Would be nice to avoid overlooking it, so I'm leaving this comment.